com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs

```diff
@@ -255,6 +255,12 @@ static internal void SafeDestroy(ref RenderTexture rt)
             }
         }
 
+        static uint VolumetricFrameIndex(HDCamera hdCamera)
+        {
+            // Here we do modulo 14 because we need the enable to detect a change every frame, but the accumulation is done on 7 frames (7x2=14)
+            return hdCamera.GetCameraFrameCount() % 14;
```

Comment:
don't we have any constant value somewhere with this 7 ? Afraid to have hardcoded value hidden in a function like this.

Reply:
right now, no when the thing is accumulation frame index is computed its explicitely written frameIndex % 7